index.js

```diff
@@ -33,7 +70,16 @@ async function start() {
 You are standing on Main Street between Church and South Winooski.
 There is a door here. A keypad sits on the handle.
 On the door is a handwritten sign.`;
+
+  const foyerMessage = "You are in a foyer. Or maybe it's an antechamber. Or a vestibule. Or an entryway. Or an atrium. Or a narthex. But let's forget all that fancy flatlander vocabulary, and just call it a foyer. In Vermont, this is pronounced 'FO-ee-yurr'. A copy of Seven Days lies in a corner."
   let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
+
+if (currentRoom === 'outside'){
+  outsideRoom(answer);
+  return
```

Comment:
`return` will break you out of the function entirely here, which will prematurely end your game.